packages/__tests__/integration/integration.spec.ts

```diff
@@ -1,942 +1,948 @@
 /* eslint-disable mocha/no-skipped-tests, mocha/no-exclusive-tests, @typescript-eslint/strict-boolean-expressions, @typescript-eslint/strict-boolean-expressions */
 import { toArray } from '@aurelia/kernel';
 import { DirtyCheckProperty, IDirtyChecker } from '@aurelia/runtime';
-import { assert, getVisibleText } from '@aurelia/testing';
+import { assert, getVisibleText, eachCartesianJoin } from '@aurelia/testing';
 import { App, Product } from './app/app';
 import { Cards } from './app/molecules/cards/cards';
 import { LetDemo } from './app/molecules/let-demo/let-demo';
 import { RandomGenerator } from './app/molecules/random-generator/random-generator';
 import { $it, assertCalls, getViewModel } from './util';
+import { ComponentMode } from './app/startup';
 
 describe('app', function () {
 
-  $it('has some readonly texts with different binding modes', function ({ host }) {
-    for (let i = 0; i < 4; i++) {
-      const selector = `read-only-text#text${i}`;
-      assert.html.textContent(selector, `text${i}`, `incorrect text for ${selector}`, host);
-    }
-  });
+  eachCartesianJoin([
+    ['app', 'enhance'] as ('app' | 'enhance')[],
```

Comment:
```suggestion
    ['app', 'enhance'] as ['app', 'enhance'],
```
Just a bit simpler and a bit more accurate. Not really important though :)

Reply:
Done